assets/src/edit-story/panels/panel/title.js

```diff
@@ -0,0 +1,96 @@
+/**
+ * External dependencies
+ */
+import styled from 'styled-components';
+import PropTypes from 'prop-types';
+
+/**
+ * WordPress dependencies
+ */
+import { useContext, useCallback } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import panelContext from './context';
+import DragHandle from './handle';
+import Arrow from './arrow.svg';
+
+const Header = styled.header`
+	background-color: ${ ( { theme, isPrimary } ) => isPrimary ? theme.colors.fg.v2 : theme.colors.fg.v1 };
+	border: 0px solid ${ ( { theme } ) => theme.colors.fg.v2 };
+	border-top-width: ${ ( { isPrimary } ) => isPrimary ? 0 : '1px' };
+	color: ${ ( { theme } ) => theme.colors.bg.v2 };
+	display: flex;
+	padding: 10px 20px;
+	justify-content: space-between;
+	align-items: center;
+	position: relative;
+`;
+
+const Heading = styled.h1`
+	color: inherit;
+	margin: 0;
+	font-weight: 500;
+	font-size: 16px;
+	line-height: 19px;
+`;
+
+const Collapse = styled.button.attrs( { type: 'button' } )`
+	color: inherit;
+	width: 28px;
+	height: 28px;
+	border: 0;
+	padding: 0;
+	background: transparent;
+	display: flex; // removes implicit line-height padding from child element
+	cursor: pointer;
+
+	svg {
+		width: 28px;
+		height: 28px;
+		${ ( { isCollapsed } ) => isCollapsed && `transform: rotate(.5turn);` }
+	}
+`;
+
+function Title( { children, isPrimary, isResizable } ) {
+	const {
+		state: { isCollapsed },
+		actions: { collapse, expand, setHeight },
+	} = useContext( panelContext );
+
+	const handleHeightChange = useCallback(
+		( deltaHeight ) => setHeight( ( height ) => height + deltaHeight ),
+		[ setHeight ],
+	);
+
+	return (
+		<Header isPrimary={ isPrimary }>
```

Comment:
I think now would a good time to add some accessibility to these panels. You can see what I have done already [here](https://github.com/ampproject/amp-wp/blob/develop-stories/assets/src/edit-story/components/inspector/inspectorTabs.js#L61). I can also see more detail on accessible accordions [here](https://www.w3.org/TR/wai-aria-practices-1.1/examples/accordion/accordion.html).

Reply:
Adding the necessary `aria-` attributes sounds easy enough for now 👍

Reply:
I've more or less done what the W3 guide recommends. I don't know how important it is for IDs to be semantic (as in: is 'panel_123' as good as 'panel_size' as it's never read out loud, just linking items), but I've added that now too.